### PR TITLE
Pioneer 20b and Papa 11 time updates

### PR DESCRIPTION
**Pull request by cdobs**
papa_11

### CODE REVIEW

**Comment by cdobs** [comment]
Screenshot 2024-07-01 092547 (1)

**Comment by cdobs** [comment]
https://phideauxfishing.com/ for vessel info

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -344,10 +344,11 @@ AR82,Neil Armstrong,2024-04-01T13:20:00,2024-04-21T14:02:00,Pioneer 20 (MAB)
...
-SKQ202408S,Sikuliaq,2024-05-28T00:00:00,2024-06-15T23:59:00,Papa 11
+SKQ202408S,Sikuliaq,2024-05-29T16:01:00,2024-06-15T16:24:00,Papa 11
 ME20240528,Megalopa,2024-05-28T16:07:00,2024-05-29T06:03:00,Endurance glider 917-6 recovery
-AR84-01,Neil Armstrong,2024-05-31T00:00:00,2024-07-02T23:59:00,Irminger 11
+AR84-01,Neil Armstrong,2024-06-02T16:02:00,2024-07-02T23:59:00,Irminger 11
 KH20240609,Kahuna,2024-06-09T09:00:00,2024-06-09T23:59:00,Pioneer glider cp_379 recovery
 AT50-25,Atlantis,2024-06-10T00:00:00,2024-06-15T23:59:00,Papa glider gp_515 recovery
 KH20240616,Kahuna,2024-06-16T09:00:00,2024-06-16T23:59:00,Pioneer glider cp_387 recovery
 STP20240618,Storm Petrel,2024-06-18T14:49:00,2024-06-19T20:44:00,Endurance CSPP operations and glider deployment
+PH20240630,Phideaux,2024-06-30T10:00:00,2024-06-30T21:00:00,Pioneer 20b glider deployments
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:354]: Looks good.

File: vessel/shiplist.csv [hunk 2]
```diff
@@ -46,6 +46,7 @@ EX,R/V,Okeanos Explorer,334A,OMAO,WTDH,369888000,8835114,68.3,6.2,5.2,,Y,Y
 OD,R/V,Oscar Dyson,33OA,NOAA,WTEP,368926382,9270335,63,6.2,5.9,,Y,N
 SU,R/V,Pacific Surveyor,,,,,,,,,,,
 PE,R/V,Pelican,32PE,LUMCON,WSK3051,367652000,8517009,35.4,4.7,2.9,,Y,Y
+PH,F/V,Phideaux,,,,,,15.8,,,,Y,N
```
Comment (snwhite) [inline vessel/shiplist.csv:49]: Looks good.

**Review by snwhite — APPROVED** [review]

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:354]: all look good

File: vessel/shiplist.csv (hunk 2 above)
Comment (rtravis-whoi) [inline vessel/shiplist.csv:49]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: vocab/vocab.csv [hunk 3]
```diff
@@ -1230,6 +1230,14 @@ CP15MOAS-GL387-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387
...
+CP15MOAS-PG564,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 564,,,,0,1000
+CP15MOAS-PG564-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 564,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP15MOAS-PG564-01-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 564,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP15MOAS-PG564-02-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 564,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP15MOAS-PG564-03-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 564,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP15MOAS-PG564-04-FLORTO000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 564,3-Wavelength Fluorometer,WET Labs,ECO BB3,0,1000
+CP15MOAS-PG564-05-NUTNRM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 564,Nitrate,Satlantic,SUNA V2,0,1000
+CP15MOAS-PG564-06-PARADM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 564,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1240]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: vocab/vocab.csv (hunk 3 above)
Comment (snwhite) [reply vocab/vocab.csv:1240]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**